### PR TITLE
Seffy install docker machine flow

### PR DESCRIPTION
**Pull request by gseffy**
add flow to install docker machine on linux

### CODE REVIEW

**Comment by oritstone** [comment]
Isn't it a duplicate of #171?

**Comment by gseffy** [comment]
It is, I closed #171

**Comment by meirwah** [comment]
a better position for this should be : 
content/io/cloudslang/docker/docker_machine/install_docker_machine/

instead of :
content/io/cloudslang/docker/install_docker_machine/

File: content/io/cloudslang/docker/install_docker_machine/give_docker_machine_permission.sl [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+##################################################################################################################################################
+
+namespace: io.cloudslang.docker.install_docker_machine
+
+operation:
+  name: give_docker_machine_permission
+  inputs:
+    - host
+    - port:
+        default: "'22'"
```
Comment (meirwah) [inline content/io/cloudslang/docker/install_docker_machine/give_docker_machine_permission.sl]: If you use ssh_flow, you will save using default values (like 22 for port)

File: content/io/cloudslang/docker/install_docker_machine/give_docker_machine_permission.sl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - username
+    - password
+    - privateKeyFile:
+        default: "''"
+    - command:
+        default: "'chmod +x /usr/local/bin/docker-machine'"
```
Comment (meirwah) [inline content/io/cloudslang/docker/install_docker_machine/give_docker_machine_permission.sl]: I think we need a generic permission operation, like "give_permission" , under base/os/linux/permissions/

File: content/io/cloudslang/docker/install_docker_machine/linux_download_binary.sl [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  inputs:
+    - host
+    - port:
+        default: "'22'"
+    - username
+    - password
+    - privateKeyFile:
+        default: "''"
+    - command:
+        default: "'curl -L https://github.com/docker/machine/releases/download/v0.2.0/docker-machine_linux-amd64 > /usr/local/bin/docker-machine'"
```
Comment (meirwah) [inline content/io/cloudslang/docker/install_docker_machine/linux_download_binary.sl]: v0.2.0 should be an operation param.
same for distro : docker-machine_linux-amd64

should be using a generic download operation...

File: content/io/cloudslang/docker/install_docker_machine/osx_download_binary.sl [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  inputs:
+    - host
+    - port:
+        default: "'22'"
+    - username
+    - password
+    - privateKeyFile:
+        default: "''"
+    - command:
+        default: "'curl -L https://github.com/docker/machine/releases/download/v0.2.0/docker-machine_darwin-amd64 > /usr/local/bin/docker-machine'"
```
Comment (meirwah) [inline content/io/cloudslang/docker/install_docker_machine/osx_download_binary.sl]: same as above...

**Comment by orius123** [comment]
@gseffy can you please update from cloudslang master, so we can see that the build is passing?

**Comment by meirwah** [comment]
Is this still in progress ?

**Comment by meirwah** [comment]
@gseffy can you update again from master?

**Pull request closed without merging**